teuthology/salt.py

```diff
@@ -52,39 +53,42 @@ def __generate_minion_keys(self):
         for rem in self.remotes.iterkeys():
             minion_id = rem.hostname
             self.minions.append(minion_id)
-            log.debug("minion: ID {}".format(minion_id,))
+            log.debug('minion: ID {}'.format(minion_id,))
             # mode 777 is necessary to be able to generate keys reliably
-            # we hit this before: https://github.com/saltstack/salt/issues/31565
-            self.master_remote.run(args = [
+            # we hit this before:
+            # https://github.com/saltstack/salt/issues/31565
+            self.master_remote.run(args=[
                 'sudo',
                 'sh',
                 '-c',
-                'if [ ! -d salt ]; then\
-                mkdir -m 777 salt; fi'])
-            self.master_remote.run(args = [
+                ('if [ ! -d salt ]; then'
+                 'mkdir -m 777 salt; fi')])
```

Comment:
I'd prefer to have the closing `])` on a new line, vertically aligned with "self.master_remote.run" but that's just my cosmetic preference.